libcrux-ml-dsa/src/simd/avx2.rs

```diff
@@ -16,15 +16,18 @@ mod vector_type;
 use arithmetic::shift_left_then_reduce;
 pub(crate) use vector_type::{AVX2RingElement, Vec256 as AVX2SIMDUnit};
 
-#[cfg(not(eurydice))]
+#[cfg(hax)]
 impl Repr for AVX2SIMDUnit {
-    fn repr(&self) -> [i32; COEFFICIENTS_IN_SIMD_UNIT] {
-        let mut result = [0i32; COEFFICIENTS_IN_SIMD_UNIT];
+    fn repr(&self) -> [i32; super::traits::COEFFICIENTS_IN_SIMD_UNIT] {
+        let mut result = [0i32; super::traits::COEFFICIENTS_IN_SIMD_UNIT];
         vector_type::to_coefficient_array(self, &mut result);
         result
     }
 }
 
+#[cfg(any(eurydice, not(hax)))]
```

Comment:
do we ever have both eurydice and hax? otherwise we could just go with not(hax)

Reply:
no, but we don't want this in the regular code. But apparently we need it for extraction.

Reply:
Got it!